code_examples/sdk_examples/src/core_features/did/08_full_did_tx.ts

```diff
@@ -0,0 +1,23 @@
+import * as Kilt from '@kiltprotocol/sdk-js'
+// Just a helper to get an extrinsic
+import getExtrinsic from '../utils/getExtrinsic'
+
+export async function signAndSendDidExtrinsic(
```

Comment:
What about `signAndSubmit`? It's a term more people are familiar with.

Reply:
good point we should stick to submit. [76db79a](https://github.com/KILTprotocol/docs/pull/183/commits/76db79a5f6a52895fbd0591a82334b2736439905)